### PR TITLE
Fix corner case in context menu in the event sheet - #987 - disable command when not usable

### PR DESCRIPTION
**Pull request by blurymind**
this fixes 
https://github.com/4ian/GDevelop/issues/987
and other noticed corner cases

### CODE REVIEW

File: newIDE/app/src/EventsSheet/index.js [hunk 1]
```diff
@@ -276,6 +276,18 @@ export default class EventsSheet extends React.Component<Props, State> {
...
 
+  _selectionCanReceiveSubEvents = () => {
+    return getSelectedEvents(this.state.selection).some(event => {
+      return event.canHaveSubEvents();
+    });
+  };
+
+  _selectionCanToggleDisabled = () => {
+    return getSelectedEvents(this.state.selection).some(event => {
+      return event.getType() !== 'BuiltinCommonInstructions::Comment';
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/index.js]: Not bad, but we can do a bit better: `return event.isExecutable()` :)

File: newIDE/app/src/EventsSheet/index.js [hunk 2]
```diff
@@ -276,6 +276,18 @@ export default class EventsSheet extends React.Component<Props, State> {
     });
   };
 
+  _selectionCanReceiveSubEvents = () => {
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/index.js]: `_selectionCanHaveSubEvents` seems more similar to `canHaveSubEvents` of `event` :)

**Comment by 4ian** [comment]
Looks pretty good! Added two comments about naming and an alternative way for non-executable events.

File: newIDE/app/src/EventsSheet/index.js (hunk 1 above)
Comment (blurymind) [reply newIDE/app/src/EventsSheet/index.js]: ah nice. I didn't notice this method initially. Will amend this now

File: newIDE/app/src/EventsSheet/index.js (hunk 2 above)
Comment (blurymind) [reply newIDE/app/src/EventsSheet/index.js]: :) good point, better follows the naming convention

**Comment by blurymind** [comment]
> Looks pretty good! Added two comments about naming and an alternative way for non-executable events.
😄 thank you for reviewing this, hopefully addresses all corner cases. Let me know if you notice others

**Comment by 4ian** [comment]
Looks good to me! Thanks for adding this :)

**Comment by blurymind** [comment]
no problem 😸 this one was simple.Sorry I missed it initially

**Pull request merged**